enarx-pr-merge-event

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import enarxbot
+import sys
+
+# Initialize Github
+github = enarxbot.connect()
+
+# Take inputs from shell and grab the PR to work on.
+(repo_name, number) = sys.argv[1:3]
+repo = github.get_repo(repo_name)
+pr = repo.get_pull(number)
```

Comment:
These are going to cause network access. But we only need access to `mergeable` and `mergeable_state`, both of which are already contained in `GITHUB_EVENT_PAYLOAD`. That is, we have all the information we need to make the decision to merge or not already. We need to make only a single network call to perform the actual merge.

So let's ditch PyGithub and just do a graphql mutation for the merge.